assets/js/atomic/blocks/product-elements/button/style.scss

```diff
@@ -42,6 +33,26 @@
 		display: none;
 	}
 
+	@keyframes scrollToUp {
+		from {
+			transform: translateY(0);
+		}
+		to {
+			transform: translateY(-60%);
+		}
+	}
+
+	@keyframes scrollFromDown {
```

Comment:
I feel a bit confusing to use the word `scroll` for these animations, I usually relate "scroll" to scrolling the page, not to elements animating up or down. I would maybe name them "slideOut" and "slideIn", which also make it clear which element is visible at the beginning and which one at the end of the animation. What do you think?